contracts/tests/package.json

```diff
@@ -0,0 +1,86 @@
+{
+    "name": "@0x/contracts-tests",
+    "version": "0.0.1",
+    "engines": {
+        "node": ">=6.12"
+    },
+    "description": "Unit tests for 0x contracts",
+    "main": "lib/src/index.js",
+    "directories": {
+        "test": "test"
+    },
+    "scripts": {
+        "build": "tsc -b",
+        "build:ci": "yarn build",
+        "pre_build": "run-s compile contracts:gen generate_contract_wrappers",
+        "test": "yarn run_mocha",
```

Comment:
Could we have this take an argument that specifies which packages's tests to run? In a world where we have all our contract tests in this package, I'd like to be able to do e.g. `CONTRACT=dev-utils yarn test`. Then we could maybe point the `test` script in the `dev-utils` package here to run the tests?

Reply:
nice suggestion! I'll do it